server/routes/me.js

```diff
@@ -184,35 +184,58 @@ router.put(
     if (!errors.isEmpty()) {
       return res.status(422).json({ success: false, msg: errors.array().toString() });
     }
+
     const user = req.decoded.user;
+
     let networkObj = await network.connectToNetwork(user);
     if (!networkObj) {
       return res.status(500).json({
         success: false,
         msg: 'Failed connect to blockchain'
       });
     }
+
     let identity = user.username;
     let response;
+
     if (user.role === USER_ROLES.STUDENT) {
       response = await network.query(networkObj, 'GetStudent', identity);
     }
+
     if (user.role === USER_ROLES.TEACHER) {
       response = await network.query(networkObj, 'GetTeacher', identity);
     }
+
     if (!response.success) {
       return res.status(500).json({
         success: false,
         msg: response.msg.toString()
       });
     }
+
+    let today = new Date();
+    let thisYear = today.getFullYear();
+
     let userInfo = JSON.parse(response.msg);
     let fullName = req.body.fullName ? req.body.fullName : '';
     let phoneNumber = req.body.phoneNumber.value ? req.body.phoneNumber.value : '';
     let email = req.body.email ? req.body.email : '';
     let address = req.body.address ? req.body.address : '';
     let sex = req.body.sex ? req.body.sex : '';
     let birthday = req.body.birthday ? req.body.birthday : '';
+
+    if (birthday !== '') {
+      let year = birthday.slice(-4);
+      year = parseInt(year);
+
+      if (
+        (thisYear - year < 6 && user.role === USER_ROLES.STUDENT) ||
```

Comment:
chỉ năm thôi thì có lẽ chưa đủ, e tham khảo thêm xử lý full ngày tháng thế này xem 
https://stackoverflow.com/questions/14231381/to-check-if-age-is-not-less-than-13-years-in-javascript

Reply:
cộng với việc ở profile cũng phải giới hạn input nhập vào nữa